sunpy/net/jsoc/jsoc.py

```diff
@@ -213,13 +221,26 @@ def query(self, *query, **kwargs):
             iargs = kwargs.copy()
             iargs.update(block)
             blocks.append(iargs)
-
             return_results.append(self._lookup_records(iargs))
 
         return_results.query_args = blocks
-
         return return_results
 
+    def get_metadata(self, *query, **kwargs):
```

Comment:
This might be better named `search_metadata` seeing how it has the same interface as the search functions.